addons/im_livechat/static/src/embed/chat_window/chat_window_container_patch.js

```diff
@@ -0,0 +1,15 @@
+/* @odoo-module */
+
+import { ChatWindowContainer } from "@mail/core/common/chat_window_container";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ChatWindowContainer.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.messaging.isReady.resolve({
+            channels: [],
+            current_user_settings: {},
+        });
```

Comment:
Also, I think there is an issue here:
- `isReady` is resolved with dummy values. Let's assume another part of the code needs to call `messaging.initialize`: the promise is already resolved, the callbacks have been call with those dummy values so the second call will not update the state which is wrong.